documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -8,20 +8,20 @@
           v-if="loading"
           indeterminate
           size="16" />
-        <v-icon
+        <i
           v-else
-          :size="isMobile && 32 || 22"
-          :color="icon.color">{{ icon.class }}</v-icon>
+          class="fas fa-folder driveFolderIcon">
+          <i class="fa-hdd driveFolderContentIcon"></i>
+        </i>
       </div>
       <div class="width-full">
         <div
           v-if="!editNameMode"
           @click="openPreview()"
           class="document-title clickable hover-underline d-inline-flex"
           :title="title">
-          <div
-            v-sanitized-html="title"
-            class="document-name text-truncate ms-4">
+          <div class="document-name text-truncate ms-4">
+          <span> {{ title }} </span>
```

Comment:
Could you explain the reason to disable title sanitization ?

Reply:
> disable title sanitization

I disabled title sanitization because the folder title contains an email address and the email should not be clickable and open a new tab to send new mail.

Reply:
Sanitization is a must -have to protect user data, we need to find a way to disable sending emails without disabling sanitiation . Could you suggest another solution for the current issue ?